src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -67,13 +67,16 @@ void GcsServer::Start() {
     // Init grpc based pubsub on GCS.
     // TODO: Move this into GcsPublisher.
     inner_publisher = std::make_unique<pubsub::Publisher>(
-        /*channels=*/std::vector<
-            rpc::ChannelType>{rpc::ChannelType::GCS_ACTOR_CHANNEL,
-                              rpc::ChannelType::GCS_JOB_CHANNEL,
-                              rpc::ChannelType::GCS_NODE_INFO_CHANNEL,
-                              rpc::ChannelType::GCS_NODE_RESOURCE_CHANNEL,
-                              rpc::ChannelType::GCS_WORKER_DELTA_CHANNEL,
-                              rpc::ChannelType::RAY_ERROR_INFO_CHANNEL},
+        /*channels=*/
+        std::vector<rpc::ChannelType>{
+            rpc::ChannelType::GCS_ACTOR_CHANNEL,
+            rpc::ChannelType::GCS_JOB_CHANNEL,
+            rpc::ChannelType::GCS_NODE_INFO_CHANNEL,
+            rpc::ChannelType::GCS_NODE_RESOURCE_CHANNEL,
+            rpc::ChannelType::GCS_WORKER_DELTA_CHANNEL,
+            rpc::ChannelType::RAY_ERROR_INFO_CHANNEL,
+            rpc::ChannelType::RAY_LOG_CHANNEL,
```

Comment:
I'm surprised that it's working in CPP. I mean there is a tailing ','😀

Reply:
This might be something new in C++17? I remember this didn't work before. 
I have been wanting this for long time. It would mitigate our pains of handling code conflicts.

Reply:
Yeah, not sure when it became available!